packages/vue/src/components/authenticator.vue

```diff
@@ -438,7 +313,6 @@ const hasRouteComponent = computed(() => {
 
         <confirm-sign-in
           v-if="route === 'confirmSignIn'"
-          @confirm-sign-in-submit="onConfirmSignInSubmitI"
           ref="confirmSignInComponent"
         >
           <template #confirmSignInSlotI>
```

Comment:
Quick note below this: 

```diff
- <template #footer="{ onConfirmSignInSubmit, onBackToSignInClicked }">
+        <template #footer>
            <slot
              name="setup-totp-footer"
-              :onConfirmSignInSubmit="onConfirmSignInSubmit"
-              :onBackToSignInClicked="onBackToSignInClicked"
            >
            </slot>
          </template>
```

The scoped slots for every `#footer` have function attributes passed to it. The original intention was to make available these functions to users so they could use them to override the footer slot functions if needed. This was also undocumented, IMO should be removed. 

They will also need to be removed in each component as well. e.g. `setup-totp.vue`

```diff
           <slot
                name="footer"
-               :onBackToSignInClicked="onBackToSignInClicked" <-- needs removed
-               :onSetupTOTPSubmit="onSetupTOTPSubmit".  <-- needs removed
              >
              </slot>```